.github/workflows/docker.yml

```
@@ -48,9 +48,21 @@ jobs:
         file: ./Dockerfile-16.04
         tags: yourbase/yb_ubuntu:16.04
         push: ${{ github.event_name == 'push' }}
+    - name: Build 16.04 yb-user image
+      uses: docker/build-push-action@v2
+      with:
+        file: ./Dockerfile-yb-user-16.04
+        tags: yourbase/yb_ubuntu_yb-user:16.04
```

I'm somewhat bothered by the mixing of underscores and dashes here. Also wondering whether we can give this a more descriptive name. WDYT about:

```suggestion
        tags: yourbase/yb_ubuntu_nonroot:16.04
```